test/e2e/mock-e2e-allowlist.js

```diff
@@ -40,14 +40,13 @@ const ALLOWLISTED_URLS = [
   'https://swap.api.cx.metamask.io/networks/1/trades?destinationToken=0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48&sourceToken=0x0000000000000000000000000000000000000000&sourceAmount=1000000000000000&slippage=2&timeout=10000&walletAddress=0x5cfe73b6021e818b776b421b1c4db2474086a7e1',
   'https://swap.api.cx.metamask.io/networks/1/trades?destinationToken=0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48&sourceToken=0x0000000000000000000000000000000000000000&sourceAmount=50000000000000000000&slippage=2&timeout=10000&walletAddress=0x5cfe73b6021e818b776b421b1c4db2474086a7e1',
   'https://swap.metaswap.codefi.network/networks/1/topAssets',
+  // Vault Decryptor
+  'https://metamask.github.io/vault-decryptor',
+  'https://metamask.github.io/vault-decryptor/',
 ];
 
 const ALLOWLISTED_HOSTS = [
   'accounts.api.cx.metamask.io',
-  'metamask.github.io',
```

Comment:
removing this entry from hosts, forces us to have the remaining live requests defined (or mocked) in the allow list URL above, with the correct exact path (including version)